drf_braces/serializers/enforce_validation_serializer.py

```diff
@@ -31,13 +31,25 @@ def run_validation(self, data=empty):
             if must_validate_fields is None or field_name in must_validate_fields:
                 raise
             else:
+                self.capture_failed_field(field_name, data)
                 raise fields.SkipField(
                     'This field "{}" is being skipped as per enforce validation logic.'
                     ''.format(field_name)
                 )
 
+    def capture_failed_field(self, field_name, field_data):
+        """Hook for capturing invalid fields. This is used to track which fields have been skipped.
+        Args:
+            field_name (str): the name of the field whose data failed to validate
+            field_data (object): the data of the field that failed validation
 
-def _create_enforce_validation_serializer(serializer, strict_mode_by_default=True):
+        Returns:
+            Not meant to return anything.
+        """
+        pass
+
+
+def _create_enforce_validation_serializer(serializer, strict_mode_by_default=True, validation_serializer_mixin_class=EnforceValidationFieldMixin):
```

Comment:
the mixin is actually applied only to fields, not serializers. lets rename the variable to be a little more accurage

Reply:
👍